charts/ccd-case-document-am-api/values.preview.template.yaml

```diff
@@ -3,4 +3,4 @@ java:
   ingressHost: ${SERVICE_FQDN}
   environment:
     DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal
```

Comment:
Is`DEFINITION_STORE_HOST` still required if it has been remove from `src/main/resources/application.yaml`

Reply:
Good catch Matt. Removed in 12f847262986e742103e153e9ae8480c676fab58

Reply:
thanks update seen in 12f847262986e742103e153e9ae8480c676fab58